### DISCUSSION TITLE
Strimzi Operator With Openshift

### DISCUSSION THREAD

**Discussion by deshpande-pavan** [strimzi/strimzi-kafka-operator#7273] [Q&A]
Hi Team, 
Could you please let me know which strimzi operator version is certified to use with Openshift? 
Looking forward to your answer. 
Thanks in advance.

**Comment by scholzj** [comment]
All versions of Strimzi are available in the OpenShift Operator Hub. So you can pick which one you want to use. Normally, you should pick the latest.
